qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/update/UpdateOfferOutput.groovy

```diff
@@ -0,0 +1,33 @@
+package life.qbic.portal.portlet.offers.update
+
+import life.qbic.datamodel.dtos.business.Offer
+import life.qbic.portal.portlet.UseCaseFailure
+
+/**
+ * Output interface for the {@link life.qbic.portal.portlet.offers.update.UpdateOffer} use case
+ *
+ * @since: 1.0.0
+ */
+interface UpdateOfferOutput extends UseCaseFailure {
+
+    /**
+     * Confirms saving of the updated offer by providing
+     * the updated offer information including the assigned identifier.
+     *
+     * @param createdOffer information about the updated offer packaged in an
+     * {@link life.qbic.datamodel.dtos.business.Offer} data-transfer object
+     * @since 1.0.0
+     */
+    void updatedOffer(Offer createdOffer)
+
+    /**
+     * Transfers the calculated net price, taxes, overheads and total price
+     * to an implementing class
+     * @param netPrice The net price calculated for the requested services
+     * @param taxes The amount of taxes for the requested services
+     * @param overheads The amount of overheads for the requested services
+     * @param totalPrice The total price for the requested services, includes taxes and overheads
+     * @since 1.0.0
+     */
+    void calculatedPrice(double netPrice, double taxes, double overheads, double totalPrice)
```

Comment:
Same as above, you might consider

```suggestion
    void onOfferPriceCalculated(double netPrice, double taxes, double overheads, double totalPrice)
```

Reply:
I think that the renaming to `onXYZ` should be tackled in another PR and would like to keep it in the PR similar to for the other use cases we implemented in the past.